tests/unit/-private/element-utils-test.js

```diff
@@ -0,0 +1,33 @@
+import { module, test } from 'qunit';
+import { getScale } from 'ember-table/-private/utils/element';
+
+module('Unit | Private | element', function(hooks) {
+  hooks.beforeEach(function() {
+    /*
+     * Use an element outside the normal test harness as that harness
+     * uses scale() on the test container.
+     */
+    this.element = document.createElement('div');
+    document.body.append(this.element);
+  });
+
+  hooks.afterEach(function() {
+    this.element.remove();
+  });
+
+  test('can get the scale of a transformed element', function(assert) {
+    let div = document.createElement('div');
+    div.style.height = '4px';
+    this.element.append(div);
+
+    assert.equal(getScale(div), 1, 'scale on a simple element is correct');
+
+    div.style.transform = 'scale(0.5)';
+
+    assert.equal(getScale(div), 2, 'scale on a scaled element is correct');
+
+    div.style.height = '1.5px';
+
+    assert.equal(getScale(div), 2, 'scale on a scaled element is correct');
+  });
+});
```

Comment:
what about the "normal scenario" test ?

Reply:
This test contains three assertions:

* scale of an element with no transforms is `1`.
* scale of an element with integer height by `0.5` is `2`
* scale of an element with float height by `0.5` is `2`

The final case is what fails without the rest of this patch.